### DISCUSSION TITLE
Machine start code and G29 usage

### DISCUSSION THREAD

**Discussion by PrybarBas** [mriscoc/Ender3V2S1#915] [Q&A] [answered]
So for my machine start G-Code, it currently has multiple lines with G29 and I'm uncertain about the correct order and if they can be combined in to a single line.
G29 L0 ; load stored mesh from index 0
G29 F10 ; z fade 10mm (or layers?)
G29 A ; activate bed-mesh smilar to M420 S1

Could this all be a single line for Machine Start G-code?
G29 L0 F10 A ;load mesh index 0, z-fade 10mm and activate mesh

**Comment by mriscoc** [comment, marked as answer]
Yes it works:
```log
Send: G29 L0 F10 A
Recv: Unified Bed Leveling System v1.01 active
Recv: Mesh loaded from slot 0
Recv: Done.
[...]
Send: M420
Recv: echo:Bed Leveling ON
Recv: echo:Fade Height 10.00
```

Of course you must have a valid mesh in slot 0

**Comment by PrybarBas** [comment]
Thx,
